src/RemoteStorage/Provider.js

```diff
@@ -1,57 +1,21 @@
 import React, {
   createContext,
   useContext,
-  useEffect,
-  useMemo,
-  useState,
 } from 'react';
-import PropTypes from 'prop-types';
 
-import { stripesConnect } from '@folio/stripes/core';
+import { useRemoteStorageMappings } from '@folio/stripes-smart-components/lib/utils';
```

Comment:
I don't think we work that way with SSC. Usually we export required things there in index.js and import them `from '@folio/stripes/smart-components';`

Reply:
it's still there. Here you can read details https://github.com/folio-org/stripes/blob/master/doc/stripes-framework.md#migrating-to-v10